src/components/user-view/chat/chat.tsx

```diff
@@ -93,8 +80,7 @@ export function Chat(props: {
   );
   const messages = curDocId ? chatState.chatLogs[curDocId] : [];
   const goalHasActivities =
-    (selectedGoal?.activities && selectedGoal.activities.length > 0) ||
-    (selectedGoal?.builtActivities && selectedGoal.builtActivities.length > 0);
+    selectedGoal?.builtActivities && selectedGoal.builtActivities.length > 0;
```

Comment:
change to `selectedGoal && selectedGoal?.builtActivities && selectedGoal.builtActivities.length > 0;`
if the `builtActivities` property will be always present, just ignore this comment.

Reply:
it will always be there